### ISSUE TITLE
nvidia_device_not_found, code -6 trex

### ISSUE THREAD

**Issue by Andr65cmd** [trexminer/T-Rex#828]
Anyone knows how can i fix this ? i think its a driver issue

**Comment by Andr65cmd** [comment]
--------------20211113 15:06:12 --------------
Mining at eu1.ethermine.org:4444, diff: 4.29 G
GPU #0: Zotac RTX 3080 Ti - 86.57 MH/s, [LHR 73.0<>] [T:45/84C, P:284W, F:80%, E:305kH/W], 140/140 R:0%
GPU #1: Gigabyte RTX 3070 - 61.54 MH/s,              [T:46C, P:129W, F:70%, E:477kH/W], 108/108 R:0%
Hashrate: 148.11 MH/s, Shares/min: 2.012 (Avg. 2.009), Avg.P: 413W, Avg.E: 359kH/W
Uptime: 2 hours 4 mins 23 secs | Algo: ethash | T-Rex v0.24.6
WD: 2 hours 4 mins 24 secs, shares: 248/248 

20211113 15:06:16 ethash epoch: 453, block: 13607844, diff: 4.29 G
20211113 15:06:25 ethash epoch: 453, block: 13607845, diff: 4.29 G
20211113 15:06:26 ethash epoch: 453, block: 13607846, diff: 4.29 G
20211113 15:06:32 ethash epoch: 453, block: 13607847, diff: 4.29 G
20211113 15:06:36 ethash epoch: 453, block: 13607848, diff: 4.29 G

--------------20211113 15:06:42 --------------
Mining at eu1.ethermine.org:4444, diff: 4.29 G
GPU #0: Zotac RTX 3080 Ti - 86.62 MH/s, [LHR 73.0<>] [T:45/88C, P:286W, F:80%, E:305kH/W], 140/140 R:0%
20211113 15:06:44 WARN: NVML: can't get fan speed for GPU #0, error code 15
GPU #1: Gigabyte RTX 3070 - 61.51 MH/s,              [T:46C, P:129W, F:70%, E:477kH/W], 108/108 R:0%
20211113 15:06:44 WARN: NVML: can't get power for GPU #0, error code 15
Hashrate: 148.14 MH/s, Shares/min: 2.012 (Avg. 2.009), Avg.P: 413W, Avg.E: 359kH/W
20211113 15:06:44 WARN: NVML: can't get mem/core clock for GPU #0, error code 17
Uptime: 2 hours 4 mins 55 secs | Algo: ethash | T-Rex v0.24.6
WD: 2 hours 4 mins 56 secs, shares: 248/248 

20211113 15:06:46 ethash epoch: 453, block: 13607849, diff: 4.29 G
20211113 15:06:50 WARN: Can't setup fan speed for [000700]: NVAPI_NVIDIA_DEVICE_NOT_FOUND, code -6
20211113 15:06:52 ethash epoch: 453, block: 13607850, diff: 4.29 G
20211113 15:06:54 ethash epoch: 453, block: 13607851, diff: 4.29 G
20211113 15:07:00 WARN: Can't setup fan speed for [000700]: NVAPI_NVIDIA_DEVICE_NOT_FOUND, code -6
20211113 15:07:10 WARN: Can't setup fan speed for [000700]: NVAPI_NVIDIA_DEVICE_NOT_FOUND, code -6
20211113 15:07:13 ethash epoch: 453, block: 13607852, diff: 4.29 G

--------------20211113 15:07:14 --------------
Mining at eu1.ethermine.org:4444, diff: 4.29 G
GPU #0: Zotac RTX 3080 Ti - 83.80 MH/s, [LHR 73.0<>] [T:0C], 140/140 R:0%
GPU #1: Gigabyte RTX 3070 - 61.52 MH/s,              [T:46C, P:129W, F:70%, E:481kH/W], 108/108 R:0%
Hashrate: 145.32 MH/s, Shares/min: 2.012 (Avg. 2.009), Avg.P: 128W, Avg.E: 481kH/W
Uptime: 2 hours 5 mins 25 secs | Algo: ethash | T-Rex v0.24.6
WD: 2 hours 5 mins 26 secs, shares: 248/248 

20211113 15:07:14 WARN: GPU #0: Zotac GeForce RTX 3080 Ti is idle, last activity was 32 secs ago
20211113 15:07:19 WARN: GPU #0: Zotac GeForce RTX 3080 Ti is idle, last activity was 37 secs ago
20211113 15:07:20 WARN: Can't setup fan speed for [000700]: NVAPI_NVIDIA_DEVICE_NOT_FOUND, code -6
20211113 15:07:25 WARN: GPU #0: Zotac GeForce RTX 3080 Ti is idle, last activity was 42 secs ago

**Comment by Andr65cmd** [comment]
oc settings --pl 82,48 --fan 80,70 --lock-cv 1200,1200 --cclock -50,-50 --mclock 1225,1225

**Comment by trexminer** [comment]
Usually it's a power/riser issue

**Comment by Andr65cmd** [comment]
> Usually it's a power/riser issue

i changed the riser and i am giving the gpu power form a different psu

**Comment by MiniHunterX** [comment]
the same error is happening to me after the last update
3060ti not LHR

**Comment by Andr65cmd** [comment]
after the last update on trex ? 0.24.7

**Comment by MCrigDEG** [comment]
Did you solve your problem? 
I've the same one
have changed the cable's + new riser
starting just with low settings ->Power Limit 78 and Temp Limit 77 without core clock or memory clock
After 10min up to 5h the failure comes again

**Comment by MiniHunterX** [comment]
1228 / 5000
Resultados de tradução
hello, first sorry for the delay to answer, in my case I managed to solve the problem, first I tested another miner "NBminer" to check if there was a problem with the riser, but the problem persisted, so I anticipated the maintenance of my rig and did all the disassembling process , cleaned everything mainly the pci express , then took the RTX3060 Ti and put it on my personal computer and did a test battery from stress to games , no problems , just in case I reflashed her bios and did the update , then I mounted the whole rig, I redid the hiveos on the ssd, even the version of the nvidia drivers changed from 495.44 which was removed from the hiveos drivers to the "updated" 470.84, then I replaced it on the rig, so the problem didn't happen anymore, but I believe The problem was very much the Riser's PCIe x1 card and the usb cable because after I cleaned both the problem didn't happen anymore, it's been more than 1 week since the problem didn't happen anymore. for this reason, if you have this problem, try to change mainly the usb 3.0 cable of the riser because it is not my first time this happens, from the older version of the riser to the newer version "s10" that I currently use.

**Comment by Andr65cmd** [comment]
> hello, first sorry for the delay to answer, in my case I managed to solve the problem, first I tested another miner "NBminer" to check if there was a problem with the riser, but the problem persisted, so I anticipated the maintenance of my rig and did all the disassembling process , cleaned everything mainly the pci express , then took the RTX3060 Ti and put it on my personal computer and did a test battery from stress to games , no problems , just in case I reflashed her bios and did the update , then I mounted the whole rig, I redid the hiveos on the ssd, even the version of the nvidia drivers changed from 495.44 which was removed from the hiveos drivers to the "updated" 470.84, then I replaced it on the rig, so the problem didn't happen anymore, but I believe The problem was very much the Riser's PCIe x1 card and the usb cable because after I cleaned both the problem didn't happen anymore, it's been more than 1 week since the problem didn't happen anymore. for this reason, if you have this problem, try to change mainly the usb 3.0 cable of the riser because it is not my first time this happens, from the older version of the riser to the newer version "s10" that I currently use.

How did you clean the pcie slots? With compressed air?

**Comment by MiniHunterX** [comment]
no, it could even be cleaned with an air compressor, like the PCIe and the riser itself, as they are more sensitive.
I cleaned them with a brush or brush and isopropyl alcohol, then I took a hair dryer like that and with the cold air I did a little use on the riser.

> > olá, primeiro desculpe pelo atraso para responder, no meu caso consegui resolver o problema, primeiro testei outro mineiro "NBminer" para verificar se havia um problema com o riser, mas o problema persistiu, então antecipei a manutenção do meu equipamento e fiz todo o processo de desmontagem, limpei tudo, principalmente o pci express, depois peguei o RT equipamento inteiro, refiz os hiveos no ssdd, até mesmo a versão dos drivers nvidia mudou de 495.44, que foi removido dos drivers hiveos para o 470.84 "atualizado", depois o substituí no equipamento, então o problema não aconteceu mais, mas acredito que o problema foi muito a placa PCIe x1 do Riser e o riser porque não é a minha primeira vez que isso acontece, da versão mais antiga do riser para a versão mais recente "s10" que eu uso atualmente.
> 
> Como você limpou os slots pcie? Com ar comprimido?
